ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/BGP_PEER_GROUPS.yml

```
@@ -39,6 +39,11 @@ bgp_peer_groups:
     # Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
     structured_config:
       description: Description for mlag_ipv4_underlay_peer via structured_config
+  mlag_ipv4_vrfs_peer:
+    name: MLAG-IPv4-OVERLAY-PEER
```

to be consistent with our keys let's do it like this
```suggestion
    name: MLAG-IPv4-VRFS-PEER
```

---

Done!